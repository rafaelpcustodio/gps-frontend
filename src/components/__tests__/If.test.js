import configureMockStore from 'redux-mock-store'
import { Map, Record } from 'immutable'
import React from 'react'
import renderer from 'react-test-renderer'

import { Provider } from 'react-redux'

import If from '../If'

describe('Tests for If component', () => {
    let store = {}
    let mockStore = {}

    it('Should render without crash', () => {
        mockStore = configureMockStore()
        store = mockStore(Record({
            gpsReducers: Map({
            })
        })())

        const tree = renderer.create(
            <Provider store={ store }>
                <If
                    children={''}
                    condition={true}
                    el={ jest.fn() }
                />
            </Provider>
        ).toJSON()

        expect(tree).toMatchSnapshot()
    })
})
