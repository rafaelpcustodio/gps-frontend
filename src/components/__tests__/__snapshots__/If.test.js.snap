// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Tests for If component Should render without crash 1`] = `""`;
